java/code/src/com/redhat/rhn/taskomatic/task/RhnQueueJob.java

```diff
@@ -49,21 +51,29 @@ public void appendExceptionToLogError(Exception e) {
 
     private void logToNewFile() {
 
+        var loggerName = this.getClass().getName();
+        final var config = ((LoggerContext) LogManager.getContext(false)).getConfiguration();
+        for (var appender : config.getLoggerConfig(loggerName).getAppenders().values()) {
+            appender.stop();
+            config.getLoggerConfig(loggerName).removeAppender(appender.getName());
```

Comment:
I would move this to the job end too... but it is more complex to figure out where this happens. In the mean time it's good here.